src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -18,10 +19,10 @@ namespace System.Text.Encodings.Web
     /// and &gt;), even if the filter provided in the constructor allows such characters.
     /// Once constructed, instances of this class are thread-safe for multiple callers.
     /// </remarks>
-    public unsafe sealed class HtmlEncoder : IHtmlEncoder
+    public unsafe sealed class HtmlEncoderOld : IHtmlEncoder
```

Comment:
These are types equivalent to what ASP.NET uses. They will be completely removed at some point, but I want to do it after completely transitioning APS.NET to new APIs and after ensuring new peft is equivalent (or better) to the old APIs.